Move RStudio keybindings to a section to assist migration (#69)
posit-dev/positron-website: keyboard-shortcuts.qmd

| <kbd>Cmd/Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>E</kbd> | Check the current R package, if any | 
| <kbd>Cmd/Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>D</kbd> | Document the current R package, if any | 

## RStudio keybindings

> If you'd prefer to use RStudio keybindings, see [this section](rstudio-keybindings.qmd) for
> instructions on how to enable them for Positron.

Reviewer: ```suggestion
If you'd prefer to use RStudio keybindings, see [our instructions](rstudio-keybindings.qmd) on how to enable them for Positron.
```
